pkgs/gnolang/helpers.go

```diff
@@ -424,6 +401,25 @@ func X(x interface{}, args ...interface{}) Expr {
 	return Nx(expr)
 }
 
+const (
+	DGTS = `(?:[0-9]+)`
+	HExX = `(?:0[xX][0-9a-fA-F]+)`
+	PSCI = `(?:[eE]+?[0-9]+)`
+	NSCI = `(?:[eE]-[1-9][0-9]+)`
+	ASCI = `(?:[eE][-+]?[0-9]+)`
+)
+
+var isIntRegex, _ = regexp.Compile(
```

Comment:
Just a nitpick here, you can use `MustCompile` instead of `Compile`, since this is a verified expression and you know it won't error out

Reply:
Same for `isFloatRegex`